### PR TITLE
[CHEC-576] Closes #51 Feature/about page

### LINKED ISSUE #51: About page on home banner = 404
https://commercejs-demo-store.netlify.app/about ==> 404.
Either create new page or redirect to commercejs.com #

Same thing happens when clicking on Find out More --> perhaps link to instagram?

### PR DESCRIPTION
**Pull request by jaepass**
- Added a simple about page with links to docs, dev resource, netlify deploy, gb repo, blog (to be posted)
- Separated social media banner and footer in order to only include footer in about page

### CODE REVIEW

**Comment by eufandem** [comment]
@jaepass can you please hyperlink commercejs.com at the end of the intro blurb (opens in new window) (all links to /docs & /resources should open in new windows)

File: components/common/Footer.js [hunk 1]
```diff
@@ -1,231 +1,133 @@
...
+      <div className="row">
+        <div className="col-12 col-sm-6 col-md-4">
+          <p className="font-family-secondary font-size-display1 mb-4">
+            Commerce.js
+          </p>
+          <div className="d-flex font-color-medium mb-5 pb-3 pb-md-0 mb-md-0">
+            <div className="pr-5">
+              <a
+                href="https://commercejs.com/docs/"
+                className="mb-3 d-block font-color-medium"
```
Comment (robbieaverill) [inline components/common/Footer.js:16]:
```suggestion
                className="mb-3 d-block font-color-medium"
                target="_blank"
```

File: components/common/Footer.js [hunk 2]
```diff
@@ -1,231 +1,133 @@
...
+              >
+                Documentation
+              </a>
+              <a
+                href="https://commercejs.com/features"
+                className="d-block font-color-medium"
```
Comment (robbieaverill) [inline components/common/Footer.js:22]:
```suggestion
                className="d-block font-color-medium"
                target="_blank"
```

File: components/common/Footer.js [hunk 3]
```diff
@@ -1,231 +1,133 @@
...
-                  <img src="/icon/arrow-long-right.svg" />
-                </a>
-              </div>
+                Features
+              </a>
+            </div>
+            <div>
+              <a
+                href="https://commercejs.com/company/about"
+                className="mb-3 d-block font-color-medium"
```
Comment (robbieaverill) [inline components/common/Footer.js:30]:
```suggestion
                className="mb-3 d-block font-color-medium"
                target="_blank"
```

File: components/common/Footer.js [hunk 4]
```diff
@@ -1,231 +1,133 @@
...
+              >
+                About
+              </a>
+              <a
+                href="http://slack.commercejs.com/"
+                className="d-block font-color-medium"
```
Comment (robbieaverill) [inline components/common/Footer.js:36]:
```suggestion
                className="d-block font-color-medium"
                target="_blank"
```

File: components/common/Footer.js [hunk 5]
```diff
@@ -1,231 +1,133 @@
...
+              href="https://app.netlify.com/start/deploy?repository=https://github.com/chec/commercejs-nextjs-demo-store"
+              className="font-color-brand font-size-caption text-uppercase text-center"
+            >
+              Contribute
+            </a>
+          </div>
+          <div className="font-color-brand font-size-caption py-4 text-right">
+          <a
+              href="https://commercejs.com/"
+              className="font-color-brand font-size-caption text-uppercase text-center"
```
Comment (robbieaverill) [inline components/common/Footer.js:122]:
```suggestion
              className="font-color-brand font-size-caption text-uppercase text-center"
              target="_blank"
```

File: components/common/SocialMedia.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+      const scrolledRatio =
+        (window.innerHeight - dimensions.top) / window.innerHeight;
+
+      this.images.forEach((image, index) => {
+        const translateRatio = imagesInfo[index] ? imagesInfo[index].translateRatio : 0;
+        image &&
+          (image.style.transform = `translateY(${scrolledRatio * translateRatio}px)`);
+      });
+    }
+  };
```
[the remark this answers is not in the document]
Comment (robbieaverill) [reply components/common/SocialMedia.js:63]: Still relevant

**Comment by robbieaverill** [comment]
Needs a rebase now

File: components/common/Footer.js (hunk 3 above)
Comment (robbieaverill) [reply components/common/Footer.js:30]: None of my suggestions appear to be resolved

Comment (jaepass) [reply components/common/Footer.js:30]: all the footer links changes were made in this PR https://github.com/chec/commercejs-nextjs-demo-store/pull/60/files which i should have just done in this PR instead. ran into conflicts while rebasing this branch so left the `Footer` component as is

Comment (jaepass) [reply components/common/Footer.js:32]: i think this fixes it, rebased and recommitted with new changes hoping will overwrite `Footer` from the last PR

**Pull request merged**